src/data/nat/totient.lean

```diff
@@ -6,6 +6,7 @@ Authors: Chris Hughes
 import algebra.big_operators.basic
 import data.nat.prime
 import data.zmod.basic
+import ring_theory.multiplicity
```

Comment:
I am not sure this import is reasonable, let me know if it's better to move the lemma somewhere else.

Reply:
After a pretty un-scientific look at the import graph (searchable version at https://alexjbest.github.io/mathlib-import-graph/) I'd say its probably Okish, it looks like the overlap of what they both import is pretty large already

Reply:
`ring_theory.multiplicity` is rather basic, it is also imported by `data.nat.multiplicity`.